packages/restitution/src/Restitution.tsx

```diff
@@ -21,7 +21,7 @@ const RestitutionValues = ({ values }: Pick<RestitutionProps, 'values'>) => {
 
 type RestitutionProps = ComponentPropsWithoutRef<'dl'> & {
   label: string;
-  value?: string;
+  value?: string | React.ReactNode;
```

Comment:
```suggestion
  value?: string | ReactNode;
```
Et donc faire l'import nécessaire sans passer par le global React

Reply:
Pris en compte, merci !